components/navbar/navbar.jsx

```diff
@@ -736,11 +768,11 @@ export default class Navbar extends React.Component {
         );
     }
 
-    createCollapseButtons = (currentId) => {
-        var buttons = [];
```

Comment:
let or const instead of vars?

Reply:
Thanks and good catch.  Also did some clean up.